tools/jenkins/run_performance.sh

```diff
@@ -34,4 +34,5 @@ set -ex
 # Enter the gRPC repo root
 cd $(dirname $0)/../..
 
-tools/run_tests/run_performance_tests.py -l c++ node ruby csharp python --netperf --category smoketest
+tools/run_tests/run_microbenchmark.py -c summary --diff_perf origin/$ghprbTargetBranch
```

Comment:
I think it does? @ctiller please correct me if I'm wrong. 

The idea behind this change is that the performance PR test results were mostly being ignored. This new performance test will show a diff in performance between a PR and the master branch and this is supposed to offer more valuable data than what the current performance PR test. This was decided by the C core team, so there definitely should be more input before this change is made. Maybe run a separate performance test for wrapper languages?